src/components/Calculator.tsx

```diff
@@ -241,7 +241,7 @@ class Calculator extends React.Component<Props, State> {
               <span>
                 Zijn er rondom uw woning vaste verhogingen aanwezig
                 (bijvoorbeeld een glazen of betonnen verhoging van een kade
-                nabij water)?
+                nabij water)?{" "}
```

Comment:
Another option is to use this html code "&nbsp;" to represent space in html but your solution is also good :)

Reply:
I think you mean

`&nbsp;`
or
`&#160;`

Adding just " " would show 
`" "` on the page, instead of a space.
With the curly braces, it is shown as a space here.

Reply:
Ah I mean `&nbsp` :)